src/OWASPZAPDotNetAPI/OWASPZAPDotNetAPI.Samples/AuthenticatedScanWithFormsAuthentication.cs

```diff
@@ -33,7 +13,7 @@ namespace OWASPZAPDotNetAPI.Samples
     class AuthenticatedScanWithFormsAuthentication
     {
         private static string _target = "http://localhost:8020/SqliModernApp";
-        private static string _apikey = string.Empty;
+        private static string _apikey = "vufbko8sihdfl5502df3863erg";

```

Comment:
This is a sample file that shows how to consume the dot net api client. Is there a better way to not hardcode it in the code (apart from putting in an application configuration file?)?